packages/eas-cli/src/utils/paths.ts

```diff
@@ -25,6 +25,9 @@ function dotExpoHomeDirectory(): string {
 
 const getStateJsonPath = (): string => path.join(dotExpoHomeDirectory(), 'state.json');
 
+export const getEasBuildRunCacheDirectoryPath = (): string =>
```

Comment:
do not mix export types, either add it to the block at the end or change all exports in this file  to `export const = ...`
I prefer the second option